code/clean/clean_eeoc.do

```diff
@@ -0,0 +1,99 @@
+/*******************************************************************************
+
+Cleans EEOC data 
+
+*******************************************************************************/
+
+import delimited "$raw_data/EEOC/cases_edited.csv", varnames(1) clear
+
+	
+/*******************************************************************************
+Rename Variables
+*******************************************************************************/
+
+ren casename resp_org
+ren civilactionnumber Civil_Action_Number
+ren court court_name
+ren courtfilingdate Court_Filing_Date_temp
+ren resolutiondate Resolution_Date_temp
+ren relief Relief_temp
+ren allegations basis
+
+ren *, lower
+
+gen state = "Federal"
+gen juris = "Employment"
+
+
+/*******************************************************************************
+Identify SH cases
+*******************************************************************************/
+
+g sh = regexm(lower(basis), "sexual harassment")
+
+g sex_cases = regexm(lower(basis), "title vii / sex‐female")
+
+// // Total cases of Sexual Harassment
+count if sh == 1
+// // Total cases brought under Title VII / Sex-Female
+count if sex_cases == 1
+
+
+
+/*******************************************************************************
+Clean columns
+*******************************************************************************/
+
+// convert 'Court Filing Date' and 'Resolution Date' to datetime format
+
+gen court_file_date = date(court_filing_date_temp, "MDY"), after(court_filing_date_temp)
+gen court_res_date = date(resolution_date_temp, "MDY"), after(resolution_date_temp)
+
+// Calculate the duration in days between filing and resolution
+gen duration = court_res_date - court_file_date
+
+// Remove the dollar sign and commas from the 'Relief' column
+replace relief_temp = regexreplaceall(relief_temp, "[$,]", "")
+
+// Create a new column to indicate missing relief
+gen missing_relief = 1 if missing(relief_temp)
```

Comment:
What is relief_temp? In 55-63 I feel like we're creating a bunch of vars unnecessarily? Why can't we just: 

g missing_relief = (relief ==0)

is that going to do everything in those 10 lines?

Reply:
One other thing these lines do is convert the relief values to numbers from strings. I did remove the extra temporary value though and clean it up.